src/backend/db/products.js

```diff
@@ -0,0 +1,163 @@
+import { v4 as uuid } from "uuid";
+
+import atomicHabits from "../../images/atomic-habits.jpg"
+import fiveAmClub from "../../images/5am-club.jpg"
+import alchemist from "../../images/alchemist.jpg"
+import doTheEpicShit from "../../images/do-the-epic-shit.jpg"
+import ikigai from "../../images/ikigai.jpg"
+import thinkAndGrowRich from "../../images/think-and-grow-rich.png"
+import theHouseOfStrangeStories from "../../images/TheHouseOfStrangeStories.jpg"
+import HorrorsOfTheNight from "../../images/HorrorsOfTheNight.jpg"
+import mahabharata  from "../../images/mahabharata.webp"
+import unfinished  from "../../images/Unfinished.webp"
+import agniKiUdaan from "../../images/agniKiUdaan.webp"
+import theFitnessCurrency from "../../images/theFitnessCurrency.webp"
+
```

Comment:
Suggestion: instead of importing here you can import all images file in one file and export from there, that your code file looks clean.

Reply:
thankyou for your suggesstion ,i have imported them into a new file